CHANGELOG.md

```diff
@@ -9,6 +9,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Fixed
 
 ### Changed
+- Changed `add_viadot_metadata_columns` decorator that will be used as a decorator for `to_df` class methods (in source or task).
```

Comment:
In this version it is for a function where DF is generated, not exactly '`to_df()`. -> or `to_df or function where DataFrame is generated.`

Reply:
Changed.